src/components/UploadFile/UploadFile.tsx

```diff
@@ -45,8 +45,10 @@ export const SaveUpload = () => {
       className="rounded m-6 border-dashed border-drg-primary-700 border-2 p-6 text-center cursor-pointer select-none hover:bg-drg-primary-400 hover:text-slate-900 transition-all duration-200"
     >
       <input {...getInputProps()} />
-      <p>Drag 'n' drop your save here, or click to select files</p>
+      <p>{`Drag 'n' drop your save here, or click to select files`}</p>
```

Comment:
Why?

Reply:
The linter has asked for it **"react/no-unescaped-entities"**

Reply:
And where is the entity? Eslint doesn't fires for me